editor/editor_log.cpp

```diff
@@ -281,7 +283,35 @@ void EditorLog::_add_log_line(LogMessage &p_message, bool p_replace_previous) {
 		log->pop();
 	}
 
-	log->add_text(p_message.text);
+	// Search for resource paths
+	Array resource_paths = resource_path_regex.search_all(p_message.text);
+	if (!resource_paths.is_empty()) {
+		// For each resource path, check if file exists and add meta tag.
+		int start = 0;
+		for (int i = 0; i < resource_paths.size(); i++) {
+			Ref<RegExMatch> current_path = static_cast<Ref<RegExMatch>>(resource_paths[i]);
+			String path_str = current_path->get_string(0);
+			String path_base = path_str.split(":")[1]; // Get file path without line numbers, in case it has them.
+
+			log->add_text(p_message.text.substr(start, current_path->get_start(0) - start));
+			if (EditorFileSystem::get_singleton()->find_file("res:" + path_base, nullptr)) {
+				log->push_meta(current_path->get_string(0));
+				log->add_text(current_path->get_string(0));
+				log->pop();
+			} else {
+				log->add_text(current_path->get_string(0));
```

Comment:
```suggestion
				log->push_meta(path_str);
				log->add_text(path_str);
				log->pop();
			} else {
				log->add_text(path_str);
```

Easier to read since we already have a dedicated variable :)

Reply:
Thanks!